Crypto_bot_pkg/crypto_selection.py

```diff
@@ -7,3 +8,22 @@
 def which_crypto():
     print("This function will allow you to choose between BTC, ETH, ADA or DOGE coins")
 
+=======
```

Comment:
Here is where the assignment from this week begins - all other file updates were to allow the installation of the package

Reply:
You could add documentation to the which_crypto() after you've made the function of course.